02/app.js

```diff
@@ -1,7 +1,22 @@
+const a = prompt('Podaj liczbę');
+const n = 9;
+let sum = 0;
 
-/* rozwiązanie z pętlą for */
-const x = 4;
+for (let i=1; i <= n; i++) {
+  sum = a * i;
+  console.log('Tabliczka mnozenia:' + sum)
+}
 
 
+let expo = 1;
+let b = prompt('Wpisz wartość')
```

Comment:
Jak już pozwalamy podawać użytkownikowi to dlaczego tylko tą liczbę?
Wtedy też przydałoby się zabezpieczyć przed "wyjątkami" :)